guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -42,38 +74,64 @@ Catalogs are compiled by a Puppet server from Puppet Manifests and data from Pup
 Candlepin:: A service within Katello responsible for subscription management.
 
 [[Compliance_policy]]
-Compliance policy:: Refers to a scheduled task executed on {ProjectServer} that checks the specified hosts for compliance against SCAP content.
+Compliance policy::
+Compliance policies refer to the application of SCAP content to hosts using {Project} with its OpenSCAP plugin.
+You can create compliance policies using the {ProjectWebUI}.
+Compliance reports require the setting of a specific XCCDF profile from a SCAP content, optionally using a tailoring file.
+You can setup scheduled tasks on {ProjectServer} that check your hosts for compliance against SCAP content.
 
 [[Compute_profile]]
 Compute profile:: Specifies default attributes for new virtual machines on a compute resource.
 
 [[Compute_resource]]
-Compute resource:: A virtual or cloud infrastructure, which {ProjectName} uses for deployment of hosts and systems.
-Examples include {oVirt}, {OpenStack}, EC2, and VMWare.
+Compute resource::
+A compute resource is an external virtualization or cloud infrastructure that you can attach to {Project}.
+{Project} can provision, configure, and manage hosts within attached compute resources.
+Examples for compute resources include VMware or libvirt and cloud providers such as Microsoft Azure or Amazon EC2.
+You can access compute resources in the {ProjectWebUI} under *Infrastructure* > *Compute Resources*.
```

Comment:
I don't think this has to be included in term definitions in general.

Reply:
Dropped.